Domains/Smartlighting/LightSchedules.md

```diff
@@ -0,0 +1,286 @@
+| `FRIDAY`      | may only trigger a switch action on Fridays                           |
+| `SATURDAY`    | may only trigger a switch action on Saturdays                         |
+| `SUNDAY`      | may only trigger a switch action on Sundays                           |
+| `WEEKDAY`     | may only trigger a switch action on weekdays (Monday to Friday)       |
+| `WEEKEND`     | may only trigger a switch action during weekends (Saturday or Sunday) |
+| `ABSOLUTEDAY` | may only trigger a switch action on the day specified in `startDay`   |
+| `ALL`         | may trigger a switch action on any day                                |
+
+## Time
+
+Each schedule entry can cause switching at a single time during the day. There are a number of ways
+in which this time can be specified, starting with `ActionTime`.
+
+| **`ActionTime`** | **Description**                                                       |
+| ----------------:| --------------------------------------------------------------------- |
+| `ABSOLUTETIME`   | a [fixed time](#fixed-time) is set for the switching moment in `Time` |
+| `SUNRISE`        | switching at sunrise at the location of the device                    |
+| `SUNSET`         | switching at sunset at the location of the device                     |
+
+For `ActionTime` values `SUNRISE` or `SUNSET` the value of `TriggerType` specifies what the actual
+switching time should be.
+
+| **`TriggerType`** | **Description**                                                                                           |
+| -----------------:| --------------------------------------------------------------------------------------------------------- |
+| `LIGHT_TRIGGER`   | [astronomical time with sensor input](#astronomical-time-with-sensor) determine the actual switching time |
+| `ASTRONOMICAL`    | the [calculated astronomical time](#astronomical-time) for sunrise or sunset is the switching time        |
+
+### Fixed Time
+
+For `ActionTime` `ABSOLUTETIME` a fixed time can be set for the switching moment as `Time`.<br>
+The `Time` value needs to be formatted in a way the [protocol implementations](#protocol-implementations-for-light-schedules)
+can handle. For the currently listed implementations, you should be fine when you use a format from:
+- `hh:mm`;
+- `hh:mm:ss`;
+- `hh:mm:ss.SSS`
+
+With `hh` from `00` to `23`, `mm` from `00` to `59`, `ss` from `00` to `59` and `SSS` from `000` to `999`.<br>
+Some protocols may accept more precise time formats than they support. The IEC61850 implementation for
+instance, will silently apply only the hours and minutes from any of the formats listed above.
+
+### Astronomical Time
+
+For `ActionTime` `SUNRISE` or `SUNSET` with `TriggerType` `ASTRONOMICAL` the calculated astronomical
+sunrise or sunset time (by the switching device, based on its longitude and latitude) will be used to
+determine the switching moment.
```

Comment:
Changed the wording according to the first of the two comments on this section.

The phrase astronomical sunrise is used to make it clear that sunrise according to the definitions with astronomical time is meant. Not a more general notion of when and what sunrise could mean.

Reply:
I prefer using the phrase astronomical sunrise and sunset. This because it actually means a calculated sunrise and sunset based on the location of the device. And I would read just sunrise and sunset as a sunrise and sunset thas was triggered by a light measurement event, not the astronomical time.

Reply:
After reading the second thread I changed my opinion. I would use Astronomical time in the paragraph and the next, Astronomical offsets. Not in the rest of the document where the trigger-type light trigger is used. There I would use time instead of astronomical time.

Reply:
@JelleHoffman's comments are in line with @bvdzwet's version. As I want to prevent further discussion, we'll stick to Bart's version.